src/modules/link/graphql/types.rs

```diff
@@ -36,6 +37,7 @@ impl From<LinkModel> for Link {
             original_url: value.original_url,
             created_at: value.created_at,
             updated_at: value.updated_at,
+            redirects: 0,
```

Comment:
I think you must pass the actual value here instead of hardcoding `0`.

```suggestion
            redirects: value.redirects,
```

Reply:
oh, thanks!